Library/Homebrew/brew.sh

```diff
@@ -276,7 +276,11 @@ then
   export HOMEBREW_RUBY_WARNINGS="-W0"
 fi
 
-export HOMEBREW_BOTTLE_DEFAULT_DOMAIN="https://homebrew.bintray.com"
+case "$(uname -s)" in
+    Darwin*)    export HOMEBREW_BOTTLE_DEFAULT_DOMAIN="https://homebrew.bintray.com";;
+    Linux*)     export HOMEBREW_BOTTLE_DEFAULT_DOMAIN="https://linuxbrew.bintray.com";;
```

Comment:
Are the `*` necessary? On my two systems, these two strings are exactly `Darwin` and `Linux` with no suffix.
A single space between `)` and `export` is sufficient.